CRM/Core/Payment/PayPalIPN.php

```diff
@@ -84,8 +85,6 @@ public function recur($input, $ids, $objects, $first) {
       return;
     }
 
-    $recur = &$objects['contributionRecur'];
```

Comment:
Since objects was not received by reference & is never used again this reference is meaningless